src/System.IO.FileSystem.AccessControl/dir.props

```diff
@@ -3,5 +3,7 @@
   <Import Project="..\dir.props" />
   <PropertyGroup>
     <AssemblyVersion>4.0.2.0</AssemblyVersion>
+    <IsNETCoreApp>true</IsNETCoreApp>
```

Comment:
@JeremyKuhne do we need this dependency? It will require us to pull in ACL's and its closure into the shared framework.